kml/serdes_gpx.cpp

```diff
@@ -209,23 +209,32 @@ void GpxParser::ParseGarminColor(std::string const & v)
   }
 }
 
+geometry::PointWithAltitude GpxParser::BuildPoint()
+{
+  geometry::PointWithAltitude point;
+  m2::PointD pt = mercator::FromLatLon(m_lat, m_lon);
+  point.SetPoint(pt);
+  point.SetAltitude(m_altitude);
```

Comment:
What about points without the altitude?

Reply:
return { mercator::FromLatLon(m_lat, m_lon), m_altitude };

Reply:
Should be reset to kInvalidAltitude - see UNIT_TEST(Gpx_Altitude_Issues)